SM/internal/services/modelDTO.go

```diff
@@ -36,15 +36,24 @@ type TaskOutput struct {
 	Movedinprogressat string
 }
 
-type MachineOutput struct {
+type MachineDTO struct {
 	ID               int64
 	Name             string
 	Isrepairrequired bool
 	Isactive         bool
 }
 
+type ShiftWorkerDTO struct {
+	Shiftid int64
+	Userid  int64
+}
+type ShiftTaskDTO struct {
+	Shiftid int64
+	Taskid  int64
+}
+
 type OutputTypes interface {
-	MachineOutput | TaskOutput | ShiftOutput | UserOutput | postgres.ShiftWorker | postgres.ShiftTask
+	MachineDTO | TaskDTO | ShiftDTO | UserDTO | ShiftWorkerDTO | ShiftTaskDTO
 }
```

Comment:
for me it is not clear why do you  need such things - monolith app could include have, or even hundreds of output models - approach with universal switch converter is not really scalabale